src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -160,60 +180,91 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
     }
   })
 
+  function getDataSetDifference(
+    oldDataSet: EntityRef[],
+    newDataSet: EntityRef[],
```

Comment:
Please rename newDataSet to changedItems or something similar to indicate what's happening